railties/lib/rails/generators/rails/app/templates/app/assets/config/manifest.js.tt

```diff
@@ -0,0 +1,8 @@
+
+<% unless options.api? -%>
+//= link_tree ../images
+<% end -%>
+<% unless options.skip_javascript -%>
+//= link_directory ../javascripts .js
```

Comment:
Do you think it is a good idea to do this? This means that all assets that are in the top level directory will be precompiled and I don't think this is a common pattern in most part of application.